src/cryptoadvance/specter/util/common.py

```diff
@@ -33,16 +34,16 @@ def replace_substring(text, start_position, replace_length, new_str):
     return text[:start_position] + new_str + text[start_position + replace_length :]
 
 
-def satamount_formatted(value):
+def format_btc_amount_as_sats(value: Union[float, str]) -> str:
```

Comment:
This is even less generic now, we can leave it like this and rename when doing the whole Decimal thing.

Reply:
The type hints are great!